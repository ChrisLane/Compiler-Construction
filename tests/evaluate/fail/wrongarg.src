// This is to test that we cannot try to access variables that don't exist from called functions.
called() {
    print $a;
}

main() {
    var x = 10;

    called(a);
}
