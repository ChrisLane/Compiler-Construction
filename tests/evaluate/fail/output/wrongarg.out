Fatal error: exception Failure("Could not find a variable during environment lookup")
